include/OpenSpaceToolkit/Astrodynamics/Trajectory/Maneuver.hpp

```diff
@@ -0,0 +1,77 @@
+/// Apache License 2.0
+
+#ifndef __OpenSpaceToolkit_Astrodynamics_Trajectory_Maneuver__
+#define __OpenSpaceToolkit_Astrodynamics_Trajectory_Maneuver__
+
+#include <OpenSpaceToolkit/Core/Types/Shared.hpp>
+
+#include <OpenSpaceToolkit/Mathematics/Objects/Vector.hpp>
+
+#include <OpenSpaceToolkit/Physics/Coordinate/Frame.hpp>
+#include <OpenSpaceToolkit/Physics/Coordinate/Position.hpp>
+#include <OpenSpaceToolkit/Physics/Coordinate/Velocity.hpp>
+#include <OpenSpaceToolkit/Physics/Time/Instant.hpp>
+
+namespace ostk
+{
+namespace astro
+{
+namespace trajectory
+{
+
+using ostk::core::types::Shared;
+
+using ostk::physics::coord::Frame;
+using ostk::physics::coord::Velocity;
+using ostk::physics::time::Instant;
+using ostk::physics::time::Duration;
+
+/// @brief                      Trajectory maneuver
+///                             Dummy Model, to be improved
+
+class Maneuver
```

Comment:
Worth making this an abstract parent, we should have impulsive and finite burn maneuvers.